pkg/minikube/constants/constants.go

```diff
@@ -81,6 +81,24 @@ func GetMinishiftHomeDir() string {
 	if ok {
 		return homeEnv
 	} else {
-		return filepath.Join(util.HomeDir(), ".minishift")
+		return filepath.Join(GetHomeDir(), ".minishift")
 	}
 }
+
+// HomeDir returns the home directory for the current user
+func GetHomeDir() string {
```

Comment:
:+1: comment still refer to `HomeDir` though :)

Reply:
Ahh.. Fixing that :-)

Reply:
Done!